e2eshark/tools/aztestsetup.py

```diff
@@ -6,8 +6,12 @@
 
 import os, sys, argparse, shutil, zipfile
 from azure.storage.blob import ContainerClient
+from azure.core.exceptions import ResourceNotFoundError
 from pathlib import Path
+from zipfile import ZipFile
 
+PRIVATE_CONN_STRING = os.environ.get("PRIVATE_CONN_STRING", default="")
```

Comment:
Environment variables can be hard to keep track of. We can have some as needed but wherever possible may be we should just have a switch with a default?

Reply:
Also, if having env variable makes sense for a particular use case, may I advise to have a naming convention that makes it easier to remember? E2ESHARK_AZ_PRIVSTORAGE_BLOBNAME -- something like that shorter but very clear what it is...

Reply:
Looks like it is a connection setup. So, may be just call it: AZ_PRIVATE_CONNECTION and keep as env. We cannot make value of it public and can pass around internally for usage.

Reply:
Done